src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,34 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Volvo");
+        manufacturer.setCountry("Sweden");
+        Manufacturer savedManufacturer = manufacturerDao.create(manufacturer);
```

Comment:
I think savedManufacturer is redundant 
```suggestion
        manufacturerDao.create(manufacturer);
```